device/esp_tinyusb/idf_component.yml

```diff
@@ -1,7 +1,7 @@
 ## IDF Component Manager Manifest File
 description: Espressif's additions to TinyUSB
 documentation: "https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/api-reference/peripherals/usb_device.html"
-version: "1.4.5"
+version: "1.5.0"
```

Comment:
There are two ways: 
- We can publish this changes and they will work with current version based on v0.15, but only in a Single IRQ mode. 
- After tinyusb will be published, it will start to use DMA